projects/noloan2/server/DataUploader.java

```diff
@@ -0,0 +1,8 @@
+package hasadna.projects.noloan2.server;
+
+public class DataUploader {
+
```

Comment:
Please reintroduce the code here, except the part inside `run()`:
https://github.com/google/startup-os/blob/master/common/Base64Tool.java#L42-L71

We'll be using it later.